src/app/client/src/app/app.component.html

```diff
@@ -81,9 +81,9 @@
     </app-onboarding-popup>
 
     <!-- popup disabled  -->
-    <div *ngIf="loadPopUps && isPopupEnabled">
+    <div *ngIf="loadPopUps && isOnboardingEnabled">
       <!-- 2nd popup BMGS  -->
-      <app-modal-wrapper #frameWorkPopUp *ngIf="showFrameWorkPopUp"
+      <app-modal-wrapper #frameWorkPopUp *ngIf="showFrameWorkPopUp && isBmgEnabled"
```

Comment:
@mithun30052001 isBmgEnabled rename to isFWSelectionEnabled

Reply:
it is changed @princegupta1131